src/svg/Painter.ts

```diff
@@ -458,7 +460,15 @@ class SVGPainter implements PainterBase {
 
     toDataURL() {
         this.refresh();
-        const html = encodeURIComponent(this._svgDom.outerHTML.replace(/></g, '>\n\r<'));
+        const outerHTML = this._svgDom.outerHTML
+            // outerHTML of `svg` tag is not supported in IE, use `parentNode.innerHTML` instead
+            // PENDING: Or use `new XMLSerializer().serializeToString(svg)`?
+            || (this._svgDom.parentNode && this._svgDom.parentNode as HTMLElement).innerHTML;
```

Comment:
`this._svgDom` can be assigned to a temporal variable `svgDom` for a smaller size.